### PR TITLE
[Loggable] Support embeddable xml mapping

### PR DESCRIPTION
**Pull request by eytanProxi**

### CODE REVIEW

File: src/Loggable/Mapping/Driver/Xml.php [hunk 1]
```diff
@@ -35,7 +35,7 @@ public function readExtendedMetadata($meta, array &$config)
 
         $xml = $xml->children(self::GEDMO_NAMESPACE_URI);
 
-        if ('entity' === $xmlDoctrine->getName() || 'document' === $xmlDoctrine->getName() || 'mapped-superclass' === $xmlDoctrine->getName()) {
+        if (in_array($xmlDoctrine->getName(), ['entity', 'document', 'mapped-superclass', 'embeddable'])) {
```
Comment (phansys) [inline src/Loggable/Mapping/Driver/Xml.php:38]:
```suggestion
        if (in_array($xmlDoctrine->getName(), ['entity', 'document', 'mapped-superclass', 'embeddable'], true)) {
```

**Review by phansys — CHANGES_REQUESTED** [review]

File: src/Loggable/Mapping/Driver/Xml.php (hunk 1 above)
Comment (eytanProxi) [reply src/Loggable/Mapping/Driver/Xml.php]: It's done

**Comment by pierre-H** [comment]
@phansys  : what should I do ?

**Comment by phansys** [comment]
Hi @pierre-H!
You should add a changelog note with the related changes ([ref.](https://github.com/doctrine-extensions/DoctrineExtensions/blob/main/CONTRIBUTING.md#changelog)) and the required tests to cover this feature.

**Pull request closed without merging**